packer/scripts/create-kube-certs.sh

```diff
@@ -2,6 +2,8 @@
 
 set -o errexit -o xtrace
 
+export PATH="${PATH}:/usr/local/bin/"
```

Comment:
This should be unnecessary, since packer runs the provisioning shell script `scripts/usr-local-bin-to-path.sh` first

Reply:
The Jenkins slave *should* get this, when built with the same packer config file as the vagrant image targets, as the provisioner which adds `/usr/local/bin` to PATH runs on all targets

Reply:
Hmm. I'll look into that some more. I am now suspecting the difference is 1) triton doesn't initially have a `vagrant` user, so I have to make packer login as root, combined with 2) my local SSH config default-enabling `ControlPath`

Reply:
From my reading, the `vagrant` user is getting added before any of the other provisioners run, so this may just be due to `ControlPath`. If there's a reason we need this explicitly added that's eluding me, a comment around the export would be helpful.

Reply:
No, because there's only one spot to tell packer what user to use. Since packer creates the `vagrant` user, it has to be run as some other user (in fact, we only have `root` available).  So there's no `su` / relogin involved, hence the problems.

Hmm. I can probably get away with just spawning a new `type: shell`; trying that…

Reply:
I still don't understand how this has been working on the other builds, but since it looks like this is inexplicably needed for Triton, I'm not opposed to leaving the PATH change

Small nit: the trailing `/` is unconventional, but it doesn't cause problems

Reply:
Ignore the above nit; as far as I can tell, there are no dangers of including a trailing `/` in `PATH`s... I know trailing `/` is typically preferred when referencing directories, so assumed there some weird compatibility reason for the convention of *not* including it in `PATH`.. as far as I can tell, there is none

Reply:
Okay, looking at this more, I see that `/etc/sudoers` has:
> `Defaults secure_path="/usr/sbin:/usr/bin:/sbin:/bin"`
(Note lack of `/usr/local/bin`, even though `/etc/login.defs` is fixed)

Trying to figure out why…

Reply:
Oh, because later in the file:
```
## Uncomment to use a hard-coded PATH instead of the user's to find commands
# Defaults secure_path="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
##

Yep, that's commented out.
```